src/metabase/sync/interface.clj

```diff
@@ -105,7 +105,7 @@
   {(s/optional-key :percent-json)   (s/maybe Percent)
    (s/optional-key :percent-url)    (s/maybe Percent)
    (s/optional-key :percent-email)  (s/maybe Percent)
-   (s/optional-key :average-length) su/PositiveNum})
+   (s/optional-key :average-length) s/Num})
```

Comment:
While sync on a local instance, I hit
```
07-30 10:07:54 ERROR sync.util :: Error running sync step: Input to save-fingerprint! does not match schema:

           [nil (named {:type {:type/Text {:average-length (not ("Number greater than zero" 0.0))}}} fingerprint)]


("sync.analyze.fingerprint$eval58243$save_fingerprint_BANG___58248.invoke(fingerprint.clj:22)"
 "sync.analyze.fingerprint$eval58269$fingerprint_table_BANG___58274$fn__58275$fn__58276$fn__58278.invoke(fingerprint.clj:57)"
 "sync.analyze.fingerprint$eval58269$fingerprint_table_BANG___58274$fn__58275$fn__58276.invoke(fingerprint.clj:47)"
 "sync.analyze.fingerprint$eval58269$fingerprint_table_BANG___58274$fn__58275.invoke(fingerprint.clj:43)"
 "sync.analyze.fingerprint$eval58269$fingerprint_table_BANG___58274.invoke(fingerprint.clj:41)"
 "sync.analyze.fingerprint$eval58442$fingerprint_fields_BANG___58447$fn__58448.invoke(fingerprint.clj:162)"
 "sync.analyze.fingerprint$eval58442$fingerprint_fields_BANG___58447.invoke(fingerprint.clj:158)"
 "sync.analyze.fingerprint$eval58468$fingerprint_fields_for_db_BANG___58473$fn__58474$fn__58475$iter__58476__58480$fn__58481$fn__58482.invoke(fingerprint.clj:172)"
 "sync.analyze.fingerprint$eval58468$fingerprint_fields_for_db_BANG___58473$fn__58474$fn__58475$iter__58476__58480$fn__58481.invoke(fingerprint.clj:171)"
 "sync.analyze.fingerprint$eval58468$fingerprint_fields_for_db_BANG___58473$fn__58474$fn__58475.invoke(fingerprint.clj:171)"
 "util.date$call_with_effective_timezone.invokeStatic(date.clj:82)"
 "util.date$call_with_effective_timezone.invoke(date.clj:71)"
 "sync.analyze.fingerprint$eval58468$fingerprint_fields_for_db_BANG___58473$fn__58474.invoke(fingerprint.clj:170)"
 "sync.analyze.fingerprint$eval58468$fingerprint_fields_for_db_BANG___58473.invoke(fingerprint.clj:165)"
 "sync.analyze$make_analyze_steps$fn__58616.invoke(analyze.clj:110)"
 "sync.util$eval48717$run_step_with_metadata__48722$fn__48726$fn__48728.invoke(util.clj:350)"
 "sync.util$with_start_and_finish_logging_SINGLEQUOTE_.invokeStatic(util.clj:98)"
 "sync.util$with_start_and_finish_logging_SINGLEQUOTE_.invoke(util.clj:92)"
 "sync.util$with_start_and_finish_debug_logging.invokeStatic(util.clj:115)"
 "sync.util$with_start_and_finish_debug_logging.invoke(util.clj:112)"
 "sync.util$eval48717$run_step_with_metadata__48722$fn__48726.invoke(util.clj:347)"
 "sync.util$eval48717$run_step_with_metadata__48722.invoke(util.clj:342)"
 "sync.util$eval48801$run_sync_operation__48806$fn__48807$fn__48808.invoke(util.clj:397)"
 "sync.util$eval48801$run_sync_operation__48806$fn__48807.invoke(util.clj:397)"
 "sync.util$eval48801$run_sync_operation__48806.invoke(util.clj:391)"
 "sync.analyze$eval58628$analyze_db_BANG___58633$fn__58634$fn__58635.invoke(analyze.clj:127)"
 "sync.util$do_with_error_handling.invokeStatic(util.clj:143)"
 "sync.util$do_with_error_handling.invoke(util.clj:138)"
 "sync.util$do_with_error_handling.invokeStatic(util.clj:141)"
 "sync.util$do_with_error_handling.invoke(util.clj:138)"
 "driver$fn__34142.invokeStatic(driver.clj:241)"
 "driver$fn__34142.invoke(driver.clj:241)"
 "driver$eval33911$fn__34072$G__33874__34081.invoke(driver.clj:57)"
 "sync.util$sync_in_context$fn__48593.invoke(util.clj:134)"
 "sync.util$with_db_logging_disabled$fn__48590.invoke(util.clj:125)"
 "sync.util$with_start_and_finish_logging_SINGLEQUOTE_.invokeStatic(util.clj:98)"
 "sync.util$with_start_and_finish_logging_SINGLEQUOTE_.invoke(util.clj:92)"
 "sync.util$with_start_and_finish_logging$fn__48579.invoke(util.clj:110)"
 "sync.util$with_sync_events$fn__48574.invoke(util.clj:84)"
 "sync.util$with_duplicate_ops_prevented$fn__48565.invoke(util.clj:63)"
 "sync.util$do_sync_operation.invokeStatic(util.clj:162)"
 "sync.util$do_sync_operation.invoke(util.clj:159)"
 "sync.analyze$eval58628$analyze_db_BANG___58633$fn__58634.invoke(analyze.clj:124)"
 "sync.analyze$eval58628$analyze_db_BANG___58633.invoke(analyze.clj:119)"
 "task.sync_databases.SyncAndAnalyzeDatabase.execute(sync_databases.clj:39)")
```

Is this change intended to fix that?

Reply:
yes https://github.com/metabase/metabase/pull/8167/files#diff-0e396461f7778c638c02e6e1f209f87c